infra/profile.py

```diff
@@ -103,8 +103,9 @@ def should_have_llvm_ir(runMethod):
 def dump_llvm_ir(runMethod, benchmark, output_directory):
```

Comment:
What is the point of `dump_llvm_ir`? It seems to move .ll files from one folder to another. Shouldn't we just put it in that folder in the first place?

Reply:
Fixed